packages/arb-token-bridge-ui/src/components/common/Layout.tsx

```diff
@@ -53,7 +53,7 @@ export function Layout(props: LayoutProps) {
       />
       <div className="relative flex flex-col sm:min-h-screen">
         <div className="flex flex-row">
-          <Sidebar />
+          <AppSidebar />
```

Comment:
<img width="954" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/441e0042-f6cb-48ab-a1a7-548cf6d9389f">
there's a z-index issue here

try it on a smaller screen width